express-postgres/server.ts

```diff
@@ -0,0 +1,88 @@
+import express from 'express';
+import { ClientError, errorMiddleware } from './lib/index.js';
+import pg from 'pg';
+
+const app = express();
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+    select "title";
```

Comment:
This SQL has a bug. Are you sure this endpoint works correctly? (You have too many trailing `;`s.)